containers/Ideas/IdeaCreate.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import { Button, TextInput, View } from 'react-native';
+import { styles } from './Idea.styles'
+
+export const IdeaCreate = props => {
+
+  const pressHandler = () => props.navigation.pop();
```

Comment:
This is to go back to the idea list once the idea is added. Later we need to use it at the same time with posting the idea